build/version.gni

```diff
@@ -1,27 +1,11 @@
-# Copyright 2015 The Chromium Authors. All rights reserved.
+# Copyright (c) 2016 Intel Corporation. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
-# This exposes the Chrome version and Crosswalk version as GN variables for
```

Comment:
Sorry, I'm still not clear with this.  My understanding: I keep the existing copyright notices, and add comments for android part. like below:
```
# Copyright 2015 The Chromium Authors. All rights reserved.
# Use of this source code is governed by a BSD-style license that can be
# found in the LICENSE file.

# This exposes the Chrome and Crosswalk version as GN variables for use in
# build files. This also calculates xwalk_version_code for Crosswalk-Android.
```
Am I correct?